recipes/taocpp-taopq/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "20210727":
```

Comment:
Won't we switch to `cci.20210727` format for newer releases?

Reply:
True! Thanks!